docs/ROADMAP.md

```diff
@@ -52,53 +52,67 @@ scenarios like limiting the customization and validation to a subset of
 allowed functions.  The function catalog has received additional functions, 
 examples and help. Please visit the [function catalog] for further information.
 
-## In Progress
-
 ### 7. Targeting resources in `kpt fn` commands
 
-Users want to invoke a kpt function (imperatively and declaratively) on a subset of 
-resources in the package by selecting them on the basis of GVKNN, package-path, 
-file path etc. For example, set work-load identity annotation on all Kubernetes 
+Users want to invoke a kpt function (imperatively and declaratively) on a subset of
+resources in the package by selecting them on the basis of GVKNN, package-path,
+file path etc. For example, set work-load identity annotation on all Kubernetes
 Service Account resources in this package [Tracker](https://github.com/GoogleContainerTools/kpt/issues/2015).
 **Estimated release date:** Last week of September 2021.
 
+## In Progress
 
-### 8. Integrate kpt with Cloud Code
-
-One of the major areas of investment is to integrate [Cloud Code](https://cloud.google.com/code) with kpt to provide 
-package authoring assistance. Users can author Kptfile and functionConfig files with
-features like auto-complete and error detection. This significantly improves the 
-discoverability of Kptfile schema, catalog functions and their functionConfigs.
-**Estimated release date:** Last week of September 2021 for Kptfile schema integration,
-function catalog integration will be released by mid of November 2021.
+### 8. Metrics for kpt
 
-### 9. Pre/Post processing step in `kpt fn render`
+We need a way to measure the adoption of kpt. This will help us understand the current
+impact of kpt and help us identify the areas where we can improve. Measuring KCC resources under
```

Comment:
This is an internal roadmap item that I am not sure will make sense to the outside people.  Also if someone asks - how do I measure the kpt usage in my org I am not sure we are accounting for that scenario.  I suggest yanking it out.

Reply:
Agree. Done.